src/Compilers/CSharp/Test/Semantic/Semantics/NullableReferenceTypesTests.cs

```diff
@@ -48735,53 +48893,53 @@ void Test1(CL1 x1, CL1? z1)
     {
         var y1 = new { p1 = x1, p2 = z1 };
         x1 = y1.p1 ?? x1;
-        x1 = y1.p2;
+        x1 = y1.p2; // 1
```

Comment:
Why did this change, don't see any `ref` usage here.?

Reply:
Correct. I just added the `// N` comments for clarity. One diagnostic was removed in `Test10` (which does involve `ref`), but that's all.

---
In reply to: [493894972](https://github.com/dotnet/roslyn/pull/47952#discussion_r493894972) [](ancestors = 493894972)